mineinabyss-systems/src/main/kotlin/com/mineinabyss/pvp/adventure/AdventurePvpListener.kt

```diff
@@ -2,55 +2,26 @@ package com.mineinabyss.pvp.adventure
 
 import com.derongan.minecraft.deeperworld.event.PlayerAscendEvent
 import com.derongan.minecraft.deeperworld.event.PlayerDescendEvent
-import com.derongan.minecraft.deeperworld.world.section.inSectionOverlap
 import com.mineinabyss.components.playerData
-import com.mineinabyss.mineinabyss.isInHub
-import net.md_5.bungee.api.chat.ClickEvent
-import net.md_5.bungee.api.chat.TextComponent
-import org.bukkit.ChatColor
+import com.mineinabyss.mineinabyss.core.MIAConfig
 import org.bukkit.event.EventHandler
 import org.bukkit.event.Listener
-import org.bukkit.event.player.PlayerMoveEvent
 
 class AdventurePvpListener : Listener {
     @EventHandler
-    fun PlayerMoveEvent.promptPvpSelect() {
-        val loc = player.location
+    fun PlayerDescendEvent.onEnterAbyss() {
```

Comment:
Function name should include what it does, not just what triggers it! The previous name sounded better.